Packs/Arcanna/Integrations/ArcannaAI/ArcannaAI.py

```diff
@@ -13,342 +16,411 @@
+		url_suffix = f"/api/v1/jobs/{job_id}/labels"
+		raw_response = requests.get(url=self.base_url + url_suffix, headers=self.get_headers(), verify=self.verify)
+		if raw_response.status_code not in [200, 422, 500]:
+			 raise Exception(f"Error HttpCode={raw_response.status_code} text={raw_response.text} Job ID :#{self.base_url}# #{url_suffix}# #{self.get_headers()}# #{job_id}# #{body}#")
+		return raw_response.json() 
+
+	def send_raw_event(self, job_id, severity, title, raw_body,id_value=None):
+		url_suffix = 'api/v1/events/'
+		if id_value is not None and id_value != "":
+			url_suffix = f"{url_suffix}{id_value}"
+		raw = json.loads(raw_body)
+		body = self.map_to_arcanna_raw_event(job_id, raw, severity, title)
+		raw_response = None
+		raw_response = requests.post(url=self.base_url + url_suffix, headers=self.get_headers(), verify=self.verify,
+									 json=body)
+		if raw_response.status_code not in [201,422,500]:
+			raise Exception(f"Error HttpCode={raw_response.status_code} text={raw_response.text} Job ID :#{self.base_url}# #{url_suffix}# #{self.get_headers()}# #{job_id}# #{body}#")
+		
+		return raw_response.json()
+
+	def map_to_arcanna_raw_event(self, job_id, raw, severity, title):
+		body = {
+			"job_id": int(job_id),
+			"title": title,
+			"raw_body": raw
+		}
+		if severity is not None:
+			severity = int(float(severity))
+			body["severity"] = severity
+		return body
+
+	def get_event_status(self, job_id, event_id, retries=10, seconds_per_retry=3):
+		retry_count = 0
+		while(retry_count < retries):
+			url_suffix = f"api/v1/events/{job_id}/{event_id}"
+			full_url = f"{self.base_url}{url_suffix}"
+			raw_response = requests.get(url=full_url, headers=self.get_headers(), verify=self.verify)
+			if raw_response.status_code not in [200,422]:
+				raise Exception(f"Error HttpCode={raw_response.status_code} text={raw_response.text} Job ID :#{self.base_url}# #{url_suffix}#")
+			if raw_response.json()['status'] == 'pending_inference':
+				retry_count += 1
+				time.sleep(seconds_per_retry)
+			else:
+				response = raw_response.json()
+				response['retry_count'] = retry_count
+				return raw_response.json()
+		return raw_response.json()
+
+	def send_feedback(self, job_id, event_id, username, arcanna_label, indicators):
+		url_suffix = f"api/v1/events/{job_id}/{event_id}/feedback"
+		body = self.map_to_arcanna_label(arcanna_label, username)
+		if indicators:
+			body["indicators"] = json.loads(indicators)
+		raw_response = requests.put(url=self.base_url + url_suffix, headers=self.get_headers(), verify=self.verify,
+									json=body)
+
+		if raw_response.status_code not in [200, 422, 500]:
+			raise Exception(f"Arcanna Error HttpCode={raw_response.status_code} body={raw_response.text}")
+		return raw_response.json()
+
+	@staticmethod
+	def map_to_arcanna_label(arcanna_label, username):
+		body = {
+			"cortex_user": username,
+			"feedback": arcanna_label
+		}
+		return body
+
+	def send_bulk(self, job_id, events):
```

Comment:
It this used anywhere after all the changes?

Reply:
removed